release/device-onvif-camera.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: device-onvif-camera
+version: 2.2.0
+releaseName: kamakura
+releaseStream: main
+semverBumpLevel: minor
+lts: false
+repo: 'https://github.com/edgexfoundry/device-onvif-camera.git'
+commitId: e602fd7d654e8598d334cfcbfc0e6f43d27ef328
```

Comment:
Need to be `9bac30ea06e4ebb0f0f10422a9d4eedb1be08404` for my recent change log update.